resources/views/layouts/app.blade.php

```diff
@@ -7,7 +7,7 @@
     <!-- CSRF Token -->
     <meta name="csrf-token" content="{{ csrf_token() }}">
 
-    <title>{{ config('app.name', 'Laravel') }}</title>
+    <title>{{ config('app.name', 'Boulder Goals') }}</title>
```

Comment:
Good to also modify this in the config file. app.php probably. And then you can basically just do `config('app.name')`